src/azure-cli/azure/cli/command_modules/batch/_validators.py

```diff
@@ -304,10 +304,10 @@ def validate_client_parameters(cmd, namespace):
         namespace.account_endpoint = cmd.cli_ctx.config.get('batch', 'endpoint', None)
 
     # Simple validation for account_endpoint
-    if not (namespace.account_endpoint.startswith('https://') or
+    if namespace.account_endpoint and not (namespace.account_endpoint.startswith('https://') or
             namespace.account_endpoint.startswith('http://')):
```

Comment:
```suggestion
    if namespace.account_endpoint and not (namespace.account_endpoint.startswith('https://') or
                                                                     namespace.account_endpoint.startswith('http://')):
```
https://www.flake8rules.com/rules/E128.html